src/display.rs

```diff
@@ -150,6 +151,11 @@ impl Display {
         let mut viewport_size = window.inner_size_pixels()
             .expect("glutin returns window size").to_physical(dpr);
 
+        // Disable depth mask, as the renderer never uses depth tests
+        unsafe {
+            gl::DepthMask(gl::FALSE);
+        }
```

Comment:
I'm not sure if this is the right place for this. Is there any specific reason why this isn't done at the top of the `QuadRenderer::new`?